packages/admin/dashboard/src/routes/orders/order-allocate-items/components/order-create-fulfillment-form/order-allocate-items-form.tsx

```diff
@@ -81,14 +82,19 @@ export function OrderAllocateItemsForm({ order }: OrderAllocateItemsFormProps) {
         return
       }
 
-      const promises = payload.map(([itemId, inventoryId, quantity]) =>
+      const promises = payload.map(([itemId, inventoryId, quantity]) => {
+        const item = itemsToAllocate.find((i) => i.id === itemId)
+        const variantInventoryItem = item?.variant?.inventory_items?.find(
+          (i) => i.inventory_item_id === inventoryId
+        )
```

Comment:
question: I assume these are guaranteed to be present? If not, I think we should add a safe-guard returning early.